src/components/curator-actions/BountyCardDetails.svelte

```diff
@@ -137,6 +135,14 @@
 						</section>
 					{/if}
 				</div>
+				{#if bounty.curator}
+					<button
+						class="w-full h-12 button-popup font-bold rounded-md"
+						on:click={() => {
+							curatorsDialogOpen = true;
+						}}>SHOW CURATORS</button
+					>
```

Comment:
```suggestion
						}}>
						SHOW CURATORS
				</button>
```
Would this look nicer in the code?